src/input_file.rst

```diff
@@ -1122,18 +1122,17 @@ Note that some tags, or entries for tags, are only used in certain ensembles (e.
   - Value 1: String - The residue name to apply this chemical potential.
   - Value 2: Double - The chemical potential value in degrees Kelvin (should be negative).
 
-  .. note:: 
-    - For binary systems, include multiple copies of the tag (one per residue kind).
-    - If there is a molecule kind that cannot be transfer between boxes (in PDB file the beta value is set to 1.00 or 2.00), an arbitrary value (e.g. 0.00) can be assigned to the residue name.
-
   .. code-block:: text
 
     #################################
     # Mol.  Name Chem.  Pot.  (K)
     #################################
     ChemPot   ISB     -968
 
-  - Value 1: String - The residue name to apply this chemical potential.
+
+    (MPI-GOMC Only)
+
+  - Value 1: String
```

Comment:
Complete your description: - Value 1: String - The residue name to apply this chemical potential.

Reply:
You told me not to repeat the description so I removed it.

Reply:
I told you to remove the whole line. But since you change the format that you are presenting, its better to keep value 1 and its description.